src/edn/mod.rs

```diff
@@ -542,19 +545,21 @@ fn rational_to_double(r: &str) -> Option<f64> {
 
 #[derive(Debug, PartialEq)]
 pub enum Error {
-    ParseEdnError(String),
+    ParseEdn(String),
+    Deserialize(String),
 }
 
 impl From<String> for Error {
     fn from(s: String) -> Self {
-        Error::ParseEdnError(s)
```

Comment:
Most of our Errors finish with `..Error` in other crates, is this (remove `Error` from name) the standard of the community?

Reply:
Also, what do you think of error handling like this? https://docs.serde.rs/src/serde/de/mod.rs.html#159

Reply:
I think it is redundant to have each variant with a `Error` suffix. I prefer without it, what do you think?

Example of redundancy:

```rust
EdnError::ParseEdnError(str)
EdnError::DeserializeError(str)
```

Instead of:

```rust
EdnError::ParseEdn(str)
EdnError::Deserialize(str)
```

Honestly I think I would even remove the `Edn` out of the `ParseEdn` variant

Reply:
Wow, that seems complex haha, I need sometime to undertand that :joy:

Reply:
I will resolve this, maybe I will look at this later, thanks for pointing out something that we can be based of in the future :)